### DISCUSSION TITLE
💖 molecule-gce is looking for additional maintainers

### DISCUSSION THREAD

**Discussion by ssbarnea** [ansible-community/molecule-plugins#34] [General]
State your interest, preferably ping me on irc.

**Comment by nbttmbrg** [comment]
Hello,

I will probably need that plugin very soon, since I'll update its code I (and a few members of my team) am willing to contribute.

Went to freenode but couldn't find you, when can I reach you there ?

**Comment by ssbarnea** [comment]
Read https://molecule.readthedocs.io/en/latest/contributing.html everyone moved from freenode to libera.

And you are welcomed!

**Comment by nbttmbrg** [comment]
I have an updated fork running and would be happy to become a maintainer and share it with the community

**Comment by ssbarnea** [comment]
I am happy to welcome @nbttmbrg as maintainer of this repo. Still that does not mean the others are not welcomed to join.

**Comment by ssbarnea** [comment]
@nbttmbrg Can you please fix the remaining issues?

**Comment by nbttmbrg** [comment]
> @nbttmbrg Can you please fix the remaining issues?

I need help on [create/destroy playbooks should not be copied to scenario ansible-community/molecule-plugins#18 ](https://github.com/ansible-community/molecule-plugins/issues/18)

Will update to fqcn too

**Comment by ssbarnea** [comment]
@nbttmbrg TBH, I was concerned about CI not reporting green, reported bugs can stay until someone propose a fix to them.
